portal/client-services/org.wso2.is.portal.user.client.api/src/main/java/org/wso2/is/portal/user/client/api/IdentityStoreClientServiceImpl.java

```diff
@@ -57,6 +63,12 @@
 public class IdentityStoreClientServiceImpl implements IdentityStoreClientService {
 
     private static final Logger LOGGER = LoggerFactory.getLogger(IdentityStoreClientServiceImpl.class);
+    private static final String USERNAME_CLAIM = "http://wso2.org/claims/username";
+    private static final String GROUPNAME_CLAIM = "http://wso2.org/claims/groupname";
```

Comment:
Yes, I was also using it in all simple to be consistent in domain-config.yaml file claims such as         

claimUri: "http://wso2.org/claims/givenname",       claimUri: "http://wso2.org/claims/lastname",      claimUri: "http://wso2.org/claims/middlename". But when go to bottom we have used camel case too..  In c4 based product, we have followed this to be all simple. Would be good verify at dev@